scripts/generate-google3-docs/resource-reference/overview.md

```diff
@@ -173,6 +173,10 @@ issues for {{product_name_short}}.
   <td><a href="/cloud-build/docs/">{{builder_name}}</a></td>
   <td><a href="/config-connector/docs/reference/resource-docs/cloudbuild/cloudbuildworkerpool.md">CloudBuildWorkerPool</a></td>
 </tr>
+<tr>
+  <td><a href="/quotas/docs/">Cloud Quotas</a></td>
```

Comment:
Do we care about alphabetical order here?

I do wonder if we could generate this file in future...

Reply:
I'll leave this to the next PR. The reference generator really needs a big change.